public/app/features/correlations/Forms/ConfigureCorrelationSourceForm.tsx

```diff
@@ -2,25 +2,39 @@ import { css } from '@emotion/css';
 import React from 'react';
 import { Controller, useFormContext } from 'react-hook-form';
 
-import { DataSourceInstanceSettings, GrafanaTheme2 } from '@grafana/data';
+import { DataSourceInstanceSettings, GrafanaTheme2, colorManipulator } from '@grafana/data';
 import { DataSourcePicker } from '@grafana/runtime';
 import { Card, Field, FieldSet, Input, useStyles2 } from '@grafana/ui';
 import { getDatasourceSrv } from 'app/features/plugins/datasource_srv';
 
 import { getVariableUsageInfo } from '../../explore/utils/links';
 
+import { TransformationsEditor } from './TransformationsEditor';
 import { useCorrelationsFormContext } from './correlationsFormContext';
 import { getInputId } from './utils';
 
-const getStyles = (theme: GrafanaTheme2) => ({
-  label: css`
-    max-width: ${theme.spacing(80)};
-  `,
-  variable: css`
-    font-family: ${theme.typography.fontFamilyMonospace};
-    font-weight: ${theme.typography.fontWeightMedium};
-  `,
-});
+const getStyles = (theme: GrafanaTheme2) => {
+  // show text color at 0.05% to show with alternating background colors of interactive table
+  const decomposedContrastColor = colorManipulator.decomposeColor(theme.colors.text.primary);
```

Comment:
This is non standard, but we needed a background color that will stand out and be slightly darker (light mode) or slightly lighter (dark mode) on the alternating background colors that the rows in the interactive table use. We set the background color as 0.05% of the text color to be distinctive from the background but also the text.

Reply:
Sorry, I see it now :D Not sure I was looking at something else. It's all about making the card standing out on dark rows. Makes sense 👍

Reply:
I think it's a good idea to make it stand out better, but in long term we may need to rethink the design. Having all forms in the table rows may create more, similar issue. I'll cross-link this comment in https://github.com/grafana/grafana/issues/65517